src/public/main.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let registrationData;
+let occupations;
+let stateOptions;
+let occupationList = document.getElementById('occupationList');
+let statesList = document.getElementById('statesList');
+
+const populateOccupations = () => {
+  for (let i = 0; i < occupations.length; i++) {
+    occupationList.options[i] = new Option(`${occupations[i]}`);
+  }
+};
+
```

Comment:
This function uses a for-loop to dynamically insert the **occupation** options into **occupationList**.